Automattic/newspack-custom-content-migrator
# Adds PHP constant to configure error reporting level

Introduces a PHP constant which can change the error reporting level.

## How to test
- add constant with value to `wp-config.php`:
`define( 'NEWSPACK_CUSTOM_CONTENT_MIGRATOR_ERROR_REPORTING_LEVEL', 'live' );`
- confirm that error reporting level is not changed

---
